pip/commands/wheel.py

```diff
@@ -151,7 +151,7 @@ def run(self, options, args):
 
         index_urls = [options.index_url] + options.extra_index_urls
         if options.no_index:
-            logger.info('Ignoring indexes: %s', ','.join(index_urls))
+            logger.debug('Ignoring indexes: %s', ','.join(index_urls))
             index_urls = []
```

Comment:
This helps to only get only pure JSON on stdout.
The related changes are to be consistent.
Plus I feel this does not need to be info, debug is enough. I was tempted to move in an other PR though.